sdk/hybridcompute/Azure.ResourceManager.HybridCompute/src/Generated/HybridComputeMachineData.cs

```diff
@@ -32,21 +37,135 @@ public HybridComputeMachineData(AzureLocation location) : base(location)
+        /// <summary> Specifies the operating system settings for the hybrid machine. </summary>
+        public OSProfile OSProfile { get; set; }
+        /// <summary> The provisioning state, which only appears in the response. </summary>
+        public string ProvisioningState { get; }
+        /// <summary> The status of the hybrid machine agent. </summary>
+        public HybridComputeStatusType? Status { get; }
+        /// <summary> The time of the last status change. </summary>
+        public DateTimeOffset? LastStatusChange { get; }
+        /// <summary> Details about the error state. </summary>
+        public IReadOnlyList<ResponseError> ErrorDetails { get; }
+        /// <summary> The hybrid machine agent full version. </summary>
+        public string AgentVersion { get; }
+        /// <summary> Specifies the hybrid machine unique ID. </summary>
+        public string VmId { get; set; }
+        /// <summary> Specifies the hybrid machine display name. </summary>
+        public string DisplayName { get; }
+        /// <summary> Specifies the hybrid machine FQDN. </summary>
+        public string MachineFqdn { get; }
+        /// <summary> Public Key that the client provides to be used during initial resource onboarding. </summary>
+        public string ClientPublicKey { get; set; }
+        /// <summary> The Operating System running on the hybrid machine. </summary>
+        public string OSName { get; }
+        /// <summary> The version of Operating System running on the hybrid machine. </summary>
+        public string OSVersion { get; }
+        /// <summary> The type of Operating System (windows/linux). </summary>
+        public string OSType { get; set; }
+        /// <summary> Specifies the Arc Machine's unique SMBIOS ID. </summary>
+        public string VmUuid { get; }
+        /// <summary> Machine Extensions information (deprecated field). </summary>
+        public IList<MachineExtensionInstanceView> Extensions { get; }
+        /// <summary> Specifies the Operating System product SKU. </summary>
+        public string OSSku { get; }
+        /// <summary> Specifies the Windows domain name. </summary>
+        public string DomainName { get; }
+        /// <summary> Specifies the AD fully qualified display name. </summary>
+        public string AdFqdn { get; }
+        /// <summary> Specifies the DNS fully qualified display name. </summary>
+        public string DnsFqdn { get; }
+        /// <summary> The resource id of the private link scope this machine is assigned to, if any. </summary>
+        public string PrivateLinkScopeResourceId { get; set; }
+        /// <summary> The resource id of the parent cluster (Azure HCI) this machine is assigned to, if any. </summary>
+        public string ParentClusterResourceId { get; set; }
+        /// <summary> Specifies whether any MS SQL instance is discovered on the machine. </summary>
+        public string MssqlDiscovered { get; set; }
```

Comment:
Should that be `MsSqlDiscovered`?
```suggestion
        public string MsSqlDiscovered { get; set; }
```